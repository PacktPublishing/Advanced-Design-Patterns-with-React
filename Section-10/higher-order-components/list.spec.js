import React from 'react'
import renderer from 'react-test-renderer'
import List from './list'

test('shows the gists', () => {
  const gists = [{ id: 1, description: 'description' }]
  const component = renderer.create(<List data={gists} />)
  const tree = component.toJSON()

  expect(tree).toMatchSnapshot()
})
