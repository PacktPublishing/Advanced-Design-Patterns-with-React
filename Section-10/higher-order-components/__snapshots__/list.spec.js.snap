exports[`test shows the gists 1`] = `
<ul>
  <li>
    description
  </li>
</ul>
`;
